libraries/scripting/common/src/kotlin/script/experimental/api/scriptCompilation.kt

```diff
@@ -106,6 +107,10 @@ val ScriptCompilationConfigurationKeys.implicitReceivers by PropertiesCollection
  */
 val ScriptCompilationConfigurationKeys.providedProperties by PropertiesCollection.key<Map<String, KotlinType>>() // external variables
 
+val ScriptCompilationConfigurationKeys.scriptFileLocationVariables by PropertiesCollection.key<HashMap<String, String>>()
```

Comment:
Please add javadoc/dokka comments to all new keys

Reply:
Done